KubeArmor/enforcer/bpflsm/utils.go

```diff
@@ -0,0 +1,20 @@
+package bpflsm
+
+import (
+	"strconv"
+)
+
+// GetSyscallName Function
+func getSyscallName(sc int32) string {
```

Comment:
Why cannot we export the function in the monitor one and use it? Let's add what's not available there?